velox/core/PlanNode.h

```diff
@@ -1487,9 +1487,11 @@ class WindowNode : public PlanNode {
   struct Frame {
     WindowType type;
     BoundType startType;
-    TypedExprPtr startValue;
+    std::optional<bool> isStartBoundConstant;
```

Comment:
The older representation of TypedExprPtr is reasonable in PlanNode. The Window operator can distinguish between a constant and column expr (see similar code for constant function parameters).

Reply:
Thanks for the suggestion, I was meaning to go through the constant parameter changes and modify accordingly, will update PR shortly.